### PR TITLE
update dependency on rocm/hip info command

### PR DESCRIPTION
**Pull request by PhaneeshB**
- temp fix to pass rocm device architecture using `--iree_rocm_target_chip=<arch>`

### CODE REVIEW

**Comment by stellaraccident** [comment]
Works for me.

For a more future looking approach, we should:

* add iree runtime support that lets us get the chip value for an enumerated device
* enable multi targeting and compile for a set of default supported chips

**Review by monorimet — COMMENTED** [review]
How does the bitcode path get exposed to iree-compile if we don't get it here? Is it no longer necessary to use `--iree-rocm-bc-dir=<>`?

File: shark/iree_utils/_common.py [hunk 1]
```diff
@@ -120,14 +120,8 @@ def check_device_drivers(device):
...
     elif device == "rocm":
-        try:
-            if sys.platform == "win32":
-                subprocess.check_output("hipinfo")
-            else:
-                subprocess.check_output("rocminfo")
-        except Exception:
-            return True
-
+        print(f"Device ROCM - Assuming hip / rocm libs already installed.")
```
Comment (powderluv) [inline shark/iree_utils/_common.py:123]: we don't need rocm libs.

**Comment by PhaneeshB** [comment]
> How does the bitcode path get exposed to iree-compile if we don't get it here? Is it no longer necessary to use `--iree-rocm-bc-dir=<>`?

yes, it is no longer required since IREE no comes with the necessary bc files internally.

File: shark/iree_utils/_common.py (hunk 1 above)
Comment (PhaneeshB) [reply shark/iree_utils/_common.py]: Updated.

**Review by Shukla-Gaurav — APPROVED** [review]
lgtm

**Pull request merged**